app/test_split_integer.py

```diff
@@ -1,21 +1,36 @@
+import pytest
 from app.split_integer import split_integer
 
 
 def test_sum_of_the_parts_should_be_equal_to_value() -> None:
-    pass
+    assert split_integer(17, 4) == [4, 4, 4, 5]
 
 
 def test_should_split_into_equal_parts_when_value_divisible_by_parts() -> None:
-    pass
+    assert split_integer(8, 4) == [2, 2, 2, 2]
 
 
 def test_should_return_part_equals_to_value_when_split_into_one_part() -> None:
-    pass
+    assert split_integer(8, 1) == [8]
 
 
 def test_parts_should_be_sorted_when_they_are_not_equal() -> None:
-    pass
+    assert split_integer(32, 6) == [5, 5, 5, 5, 6, 6]
 
 
 def test_should_add_zeros_when_value_is_less_than_number_of_parts() -> None:
-    pass
+    assert split_integer(2, 4) == [0, 0, 1, 1]
+
+
+def test_should_raise_error_for_type_of_value() -> None:
```

Comment:
why you add some extra tests for validation agruments? we have condition "You don't need to validate arguments (they are always valid)."